<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WndLogin</class>
 <widget class="QWidget" name="WndLogin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>768</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>768</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset resource="imandqss.qrc">
    <normaloff>:/image/image/logo.png</normaloff>:/image/image/logo.png</iconset>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>255</width>
     <height>184</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>255</width>
     <height>184</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>255</width>
     <height>184</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/image/logo.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>241</width>
     <height>391</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="sLogin">
    <widget class="QLineEdit" name="eAccount">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>180</width>
       <height>43</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>180</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>180</width>
       <height>43</height>
      </size>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">/* 矩形 5 */
position: absolute;
width: 176px;
height: 39px;
top: -68px;
left: -500px;


background: rgb(255, 255, 255);
border-radius: 20px;</string>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>输入账号</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnReg">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>190</y>
       <width>93</width>
       <height>29</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>注册账户</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ePass">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>130</y>
       <width>180</width>
       <height>43</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>180</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>180</width>
       <height>43</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">/* 矩形 5 */
position: absolute;
width: 176px;
height: 39px;
top: -68px;
left: -500px;

background: rgb(255, 255, 255);
border-radius: 20px;</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>输入密码</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnLogin">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>240</y>
       <width>180</width>
       <height>40</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>180</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>180</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>登录</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>93</x>
       <y>30</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>用户登陆</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="sReg">
    <widget class="QLineEdit" name="eUers">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>180</width>
       <height>43</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>180</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>180</width>
       <height>43</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">/* 矩形 5 */
position: absolute;
width: 176px;
height: 39px;
top: -68px;
left: -500px;


background: rgb(255, 255, 255);
border-radius: 20px;</string>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>输入用户名</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="eAccount_">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>180</width>
       <height>43</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>180</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>180</width>
       <height>43</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">/* 矩形 5 */
position: absolute;
width: 176px;
height: 39px;
top: -68px;
left: -500px;


background: rgb(255, 255, 255);
border-radius: 20px;</string>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>输入账号</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ePassword">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>160</y>
       <width>180</width>
       <height>43</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>180</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>180</width>
       <height>43</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">/* 矩形 5 */
position: absolute;
width: 176px;
height: 39px;
top: -68px;
left: -500px;


background: rgb(255, 255, 255);
border-radius: 20px;</string>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>输入密码</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ePassword_">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>220</y>
       <width>180</width>
       <height>43</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>180</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>180</width>
       <height>43</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">/* 矩形 5 */
position: absolute;
width: 176px;
height: 39px;
top: -68px;
left: -500px;


background: rgb(255, 255, 255);
border-radius: 20px;</string>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>确认密码</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnReg_">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>280</y>
       <width>81</width>
       <height>40</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>81</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>81</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>注册</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnRet">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>280</y>
       <width>81</width>
       <height>40</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>81</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>81</width>
       <height>40</height>
      </size>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>返回</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>注册信息：</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>0</y>
     <width>1371</width>
     <height>771</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2"/>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-10</y>
     <width>261</width>
     <height>781</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/image/3.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_2</zorder>
  <zorder>stackedWidget</zorder>
  <zorder>label</zorder>
  <zorder>verticalLayoutWidget</zorder>
 </widget>
 <resources>
  <include location="imandqss.qrc"/>
 </resources>
 <connections/>
</ui>
